From e99e7421b4ee5fd5eee45302038c48e7ff39d02e Mon Sep 17 00:00:00 2001
From: Volker Krause <vkrause@kde.org>
Date: Tue, 16 Nov 2021 16:42:41 +0100
Subject: [PATCH] Fix the Windows build

BER::Element is exported meanwhile, so we don't have to built it into
the VDV certificate download again.
---
 src/lib/vdv/certs/CMakeLists.txt | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/src/lib/vdv/certs/CMakeLists.txt b/src/lib/vdv/certs/CMakeLists.txt
index 8595a66..d156585 100644
--- a/src/lib/vdv/certs/CMakeLists.txt
+++ b/src/lib/vdv/certs/CMakeLists.txt
@@ -5,10 +5,9 @@ add_executable(vdv-cert-downloader
     cert-downloader.cpp
     ../vdvcertificate.cpp
     ../iso9796_2decoder.cpp
-    ../../tlv/berelement.cpp
 )
 
 target_include_directories(vdv-cert-downloader PRIVATE ${CMAKE_BINARY_DIR}/src/lib ${CMAKE_CURRENT_SOURCE_DIR}/..)
-target_link_libraries(vdv-cert-downloader PRIVATE Qt::Core OpenSSL::Crypto)
+target_link_libraries(vdv-cert-downloader PRIVATE Qt::Core OpenSSL::Crypto KPimItinerary)
 
 add_custom_target(vdv-download-certs COMMAND vdv-cert-downloader WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR})
-- 
2.34.0

